libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -265,13 +266,24 @@ public Map<String, List<MitigationRoute>> getCimitConfig() throws ConfigExceptio
     }
 
     public boolean enabled(FeatureFlag featureFlag) {
-        return Boolean.parseBoolean(
-                getSsmParameter(ConfigurationVariable.FEATURE_FLAGS, featureFlag.getName()));
+        return enabled(featureFlag.getName());
     }
 
     public boolean enabled(String featureFlagValue) {
-        return Boolean.parseBoolean(
-                getSsmParameter(ConfigurationVariable.FEATURE_FLAGS, featureFlagValue));
+        boolean enabled = false;
+        try {
+            enabled =
+                    Boolean.parseBoolean(
+                            getSsmParameter(ConfigurationVariable.FEATURE_FLAGS, featureFlagValue));
+        } catch (SsmException ex) {
```

Comment:
Is there anything more precise we can/should filter on here (i.e. so we only catch 'parameter not found' errors, and not other exceptions)?

Reply:
yeah thought so.....can use
ssmProvider.get - ParameterNotFoundException 
ssmProvider.getMultiple - InvalidFilterKeyException

Reply:
done